packages/contracts/contracts/interfaces/IVelodromeStakingRewards.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity 0.8.19;
+
+interface IVelodromeStakingRewards {
+  function deposit(uint256 amount, uint tokenId) external;
```

Comment:
where have you found this function selector?
According to this [contract](https://optimistic.etherscan.io/address/0xcc53cd0a8ec812d46f0e2c7cc5aadd869b6f0292#code) its one of these two options:
`    function deposit(uint256 _amount) external {
        _depositFor(_amount, _msgSender());
    }

    /// @inheritdoc IGauge
    function deposit(uint256 _amount, address _recipient) external {
        _depositFor(_amount, _recipient);
    }`